package.json

```diff
@@ -45,24 +47,25 @@
 		"jest": "^28.1.0",
 		"jest-environment-jsdom": "^28.1.0",
 		"lint-staged": "^12.4.1",
+		"postcss": "^8.4.14",
+		"postcss-loader": "^7.0.0",
+		"postcss-preset-env": "^7.6.0",
 		"prettier": "2.6.2",
+		"style-loader": "^3.3.1",
+		"tailwindcss": "^3.0.24",
 		"webpack": "^5.72.0",
 		"webpack-cli": "^4.9.2",
 		"webpack-dev-server": "^4.9.0"
 	},
 	"dependencies": {
-		"@emotion/react": "^11.9.0",
-		"@emotion/styled": "^11.8.1",
-		"@fontsource/roboto": "^4.5.7",
-		"@mui/icons-material": "^5.8.0",
-		"@mui/material": "^5.8.0",
 		"@reduxjs/toolkit": "^1.8.1",
 		"cors": "^2.8.5",
 		"express": "^4.18.1",
 		"react": "^18.1.0",
 		"react-dom": "^18.1.0",
 		"react-redux": "^8.0.1",
-		"react-router-dom": "^6.3.0"
+		"react-router-dom": "^6.3.0",
+		"tailwindscss": "^0.3.0"
```

Comment:
I think this dependency 'tailwindscss' is not being used therefore it should be removed but not a big deal